programs/create-genesis/main.cpp

```diff
@@ -114,6 +114,7 @@ fc::sha256 check_hash(const genesis_info::file_hash& fh) {
 
 void read_contract(const genesis_info::account& acc, contract_abicode& abicode) {
     abicode.update = acc.update && *acc.update;
+    abicode.privileged = acc.privileged.valid() ? *acc.privileged : false;
```

Comment:
it's somewhat confusing when same logic implemented differently in this two lines of code.
for me the short syntax is preferable because it's c++17 compatible (the optional contextually converted to bool, and there is no `valid()` in `std::optional`).